cartaya/2/style.css

```diff
@@ -0,0 +1,184 @@
+@font-face {
+    font-family: 'HelveticaNeueCE35Thin';
+    src: url('Helvetica Neue CE 35 Thin.ttf');
+}
+
+* { margin: 0px;}
+
+header {
+    background-color: #447CA2;
+    height: 102px;
+    margin-top: 57px;    
+}
+
+main {
+    overflow: auto;
+    padding-bottom: 50px;
+}
+
+#h1 {
+    font-family: HelveticaNeueCE35Thin;
+    font-size: 102px;
+    letter-spacing: 2px;
+    color: #FFFFFF;
+    margin-left: 53px;
+    padding-top: 7px;
+    position: absolute;
+}
+
+.color-borde {
+    border: 2pt;
+    border-style: solid;
+    border-color: #D0CCCC;
+}
+
+.lupa {
+    padding: 6px 5px;
+}
+
+#lupa {
+    width: 24px;
+    height: 27px;
+    float: left;
+    position: relative;
+}
+
+#people-search {
+    margin-top: 28px;
+    margin-left: 53px;
+}
+
+#search {
+    width: 213px;
+    height: 32.33px;
+    padding: 1px 1px 1px 2px;
+    float: left;
+    position: relative;
+    font-family: HelveticaNeueCE35Thin;

```

Comment:
You can define the font-family property in the element container, this is an Inherited property you don’t need specify in each element inside the container. If all the texts in the document have the same font, define this property only in the general element container.